security/oidc-client-mutual-tls/src/test/java/io/quarkus/ts/security/oidcclient/mtls/AuthenticationTestSuite.java

```diff
@@ -0,0 +1,39 @@
+package io.quarkus.ts.security.oidcclient.mtls;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.equalTo;
+
+import org.apache.http.HttpStatus;
+import org.junit.jupiter.api.Test;
+
+public interface AuthenticationTestSuite {
```

Comment:
I am not sure if this interface should be a class as `IncorrectKsFileTypeOidcMtlsIT`
Maybe could be a class and extends `BaseOidcMtlsIT`

Reply:
Refactored, thanks!